File: help/eng_gb/exercicios_da_ficha_4/Ficha04_010/Ficha04_010.php
<?php
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Actual number     : " ;
    $r1 = floatval(isset($_POST["r1"])? $_POST["r1"] : isset($_GET["r1"]) ? $_GET["r1"] : 0);
    
    echo $r1 . "<br/>";// print input 
    echo  "Actual number     : " ;
    $r2 = floatval(isset($_POST["r2"])? $_POST["r2"] : isset($_GET["r2"]) ? $_GET["r2"] : 0);
    
    echo $r2 . "<br/>";// print input 
    echo  "Actual Sum     : " ;
    $r3 = floatval(isset($_POST["r3"])? $_POST["r3"] : isset($_GET["r3"]) ? $_GET["r3"] : 0);
    
    echo $r3 . "<br/>";// print input 
    $sum = $r1 + $r2 ;
    $distance = abs( $sum - $r3 ) ;
    $precision = 1.0E-5 ;
    echo $r1  .  " + " . $r2  .  "     = " . $r3  .  " => " ;
    if($distance < $precision ){
        echo  "True" ;
    }else{
        echo  "False" ;
    }
    echo  "\n\n" . $r1  .  " + " . $r2  .  "     = " . ( ( $r1 + $r2 )  ) ;
    echo  "\nDistance     : " . $distance  ;
    echo  "\nPrecision     : " . $precision  ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	domingo, 2022 set 18 23:28

?>

File: help/eng_gb/exercicios_da_ficha_4/Ficha04_010/Ficha04_010.c
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//Main Function
int main() {
    printf("Actual number \t: ");
    float r1;
    scanf("%f", &r1);
    printf("Actual number \t: ");
    float r2;
    scanf("%f", &r2);
    printf("Actual Sum \t: ");
    float r3;
    scanf("%f", &r3);
    float sum = r1 + r2 ;
    float distance = abs( sum - r3 ) ;
    float precision = 1.0E-5 ;
    printf("%f + %f \t= %f => ",r1 ,r2 ,r3 );
    if(distance < precision ){
        printf("True");
    }else{
        printf("False");
    }
    printf("\n\n%f + %f \t= %f",r1 ,r2 ,( r1 + r2 ) );
    printf("\nDistance \t: %f",distance );
    printf("\nPrecision \t: %f",precision );
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	domingo, 2022 set 18 23:28
